app/Http/Controllers/Api/NominatableCollectionController.php

```diff
@@ -28,6 +28,7 @@ public function index(Request $request): JsonResponse
                                 ->searchByName($request->get('query'))
                                 ->limit(15)
                                 ->votable()
+                                ->erc721()
```

Comment:
could we set a default scope to only use erc721? and have on explicitly to include erc1155? that might avoid these issues going forward instead of having to explicitly ignore the ones we don't want to show 🤔